lib/src/local_backend.rs

```diff
@@ -307,10 +307,18 @@ pub fn commit_to_proto(commit: &Commit) -> crate::protos::local_store::Commit {
     proto.description = commit.description.clone();
     proto.author = Some(signature_to_proto(&commit.author));
     proto.committer = Some(signature_to_proto(&commit.committer));
+    proto.secure_sig = commit.secure_sig.as_ref().map(|s| s.sig.clone());
     proto
 }
 
-fn commit_from_proto(proto: crate::protos::local_store::Commit) -> Commit {
+fn commit_from_proto(mut proto: crate::protos::local_store::Commit) -> Commit {
+    // note how .take() sets the secure_sig field to None before getting the data
+    // needs to be first since proto is partially moved a bunch below
```

Comment:
I'm not sure how to read this. Maybe each line is just meant to be its own sentence?

Reply:
Yep
Made them into full sentences